includes/class-apple-news.php

```diff
@@ -261,10 +289,48 @@ public function create_default_theme() {
 			}
 		}
 
+		// Negotiate screenshot URL.
+		$theme_settings['screenshot_url'] = plugins_url(
+			'/assets/screenshots/default.png',
+			__DIR__
+		);
+
 		// Save the theme and make it active.
 		$theme->load( $theme_settings );
 		$theme->save();
 		$theme->set_active();
+
+		// Load the example themes, if they do not exist.
+		$example_themes = array(
+			'classic' => 'Classic',
```

Comment:
Do the theme names need to be translatable? Not the keys, but the display values.